requirements.txt

```diff
@@ -1,6 +1,6 @@
-ansible==2.2.2.0
```

Comment:
Agreed; actually I think it would be wise to pin the versions that we ship in the official openshift-ansible image (and change them in lockstep as much as possible). Eventually maybe customers will just use that and all this Ansible version madness will be over.